docs/en/operations/settings/settings.md

```diff
@@ -3667,6 +3667,16 @@ Possible values:
 
 Default value: `0`.
 
+## force_timezone {#force_timezone}
+
+Sets a default timezone for session. All DateTime/DateTime64 values that have no explicit timezone specified are treated as having this timezone instead of default.
```

Comment:
Sets a default timezone for session. All DateTime/DateTime64 values that have no explicit timezone specified are treated as having this timezone instead of default.
```suggestion
If specified, sets a implicit timezone (instead of server-default). All DateTime/DateTime64 values (and/or functions results) that have no explicit timezone specified are treated as having this timezone instead of default.
Examples:
:) SELECT timeZone(), timeZoneOf(now())
┌─timeZone()─┬─timeZoneOf(now())─┐
│ UTC        │ UTC               │
└────────────┴───────────────────┘

:) SELECT timeZone(), timeZoneOf(now()) SETTINGS  force_timezone = 'Europe/Amsterdam'
┌─timeZone()─┬─timeZoneOf(now())─┐
│ UTC        │ Europe/Amsterdam  │
└────────────┴───────────────────┘
```